README.md

```diff
@@ -1 +1,84 @@
+- Given a driver object, you should be able to retrieve the list of trip instances that only this driver has taken
+- Given a driver object, you should be able to retrieve an average rating for that driver based on all trips taken
+- You should be able to retrieve all drivers from the CSV file
+- You should be able to find a specific driver using their numeric ID
+
+#### Rider
+- Each rider should have information uniquely identifying them like ID, name and phone number
+- Given a rider object, you should be able to retrieve the list of trip instances that only this rider has taken
+- Given a rider object, you should be able to retrieve the list of all previous driver instances (through the trips functionality built above)
+- You should be able to retrieve all riders from the CSV file
+- You should be able to find a specific rider using their numeric ID
+
+#### Trip
+- Each trip should have information uniquely identifying it like ID, rider ID, a driver ID, date, rating
+- Each rating should be within an acceptable range (1-5)
+- Given a trip object, you should be able to retrieve the associated driver instance through the driver ID
+- Given a trip object, you should be able to retrieve the associated rider instance through the rider ID
+- You should be able to find all trip instances for a given driver ID
+- You should be able to find all trip instances for a given rider ID
+- You should be able to retrieve all trips from the CSV file
+
+## Getting Started
+This is a level 3, individual project.
+
+We will use the same project structure we used for the previous project. Classes should be in files in the `lib` folder, and tests should be in files in the `specs` folder. You should utilize a spec helper file. You will run tests by executing the `rake` command, as configured in a Rakefile. You should have **95% code coverage** using simplecov.
+
+The `support` folder contains CSV files which will drive your system design. Each CSV corresponds to a different type of object _as well as_ creating a relationship between different objects.
+
+## Baseline
+### Discussion
+#### Phase 1
+Let's start by **individually** creating the list of nouns and verbs that you would utilize to describe this problem.
+
+After a few minutes, then you'll get together with your seat squad to discuss the nouns and verbs you came up with.
+
+#### Phase 2
+Use your list of nouns to begin thinking about what classes you will want to use. Use your list of verbs associated with nouns to consider what methods you need to put in which classes.
+
+After a few minutes, then you'll get together with your seat squad to discuss the classes and methods you came up with.
+
+#### Phase 3
+Create a diagram that describes how each of these classes and methods (messages) will interact with one another as well as the CSV files.
+
+### Setup
+1. You will fork this repository in GitHub
+1. You will git clone the repository to your computer
```

Comment:
For these first two steps we should just say "Fork this.." and "Clone the..." to match the style of the following steps.